docs/source/daml/daml-studio/daml/Abort.daml

```diff
@@ -1,11 +1,16 @@
--- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 -- SPDX-License-Identifier: Apache-2.0
 
-
 module Abort where
 
-{-
-abortTest = scenario do
+import Daml.Script hiding (script)
+
+-- | This lets us type check 'abortTest' without letting it fail on `daml test`
```

Comment:
a bit hacky, but I think it's better than our current approach of keeping failing scenarios in block comments. 

If users actually want to write failing scripts to test that bad cases are indeed forbidden, we might want to add something like `scriptShouldFail`, perhaps taking an argument for (a substring of/a predicate on) the expected error message?

Reply:
yeah it has come up a few times but so far we haven’t implemented something. Part of what makes this difficult is that Daml Script also works over the gRPC API and that does not expose exceptions in a structured fashion so this gets super hacky. But I think we might be better off adding a Daml Studio only function for this than doing nothing. Anyway for your PR that seems like a fine solution :+1:

Reply:
ah yeah I see how that's tricky. Good to know it's in our radar though, thanks!